study.md

```diff
@@ -237,7 +265,25 @@ Are heuristics the same thing rules? What are some design heuristics you can use
 to improve the design of your code?
 
 ```md
-<!-- your answer here -->
+Heuristics are "rules of thumb," meaning that they are highly recommended to follow but
+not actual rules that must be followed.  It would make life much easier, however.
+
+-Hide data within its' classes
+-Functionality of your code should not rely on the user
+-Avoid clutter in the public interface of a class, don't have things that users
+won't user / aren't interested in using
+-Related data and behavior should be kept in one place
+-Avoid methods that seem to be using accessors or methods from other classes.
+If classes share methods, it will be easier to break both if changing just one
+-Role vs. class, make sure not to define a class if it is only a role
+-Beware of classes with many accessor methods
+-Overall avoid having too many ways for code to be overwritten (such as having
+too many accessors, since they allow code to be changed)
+-Avoid "god" classes, they will have too much responsibility
+
```

Comment:
There's quite a bit different here than what others posted. Good job finding more heuristics than the ones we presented to you. Awesome!